libs/wallet/provider/uphold/uphold.go

```diff
@@ -236,25 +237,31 @@ func submit(
 	if err != nil {
 		return nil, resp, fmt.Errorf("%w: %s", errorutils.ErrFailedBodyRead, err.Error())
 	}
+	sbody := string(body)
+	for _, p := range []string{`"description":\s*"[^"]+"\s*,?`, `"\w+Country":\s*"[^"]+"\s*,?`} {
```

Comment:
question: do we have an example of the response we are trying to redact this from? 

we could add an example response to a test so, I wonder if we can use an or instead of having two separate passes or the regex

Reply:
I added a test with an example object.